ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/terminal/TerminalPresenter.java

```diff
@@ -92,7 +92,7 @@ public TerminalPresenter(
    * Connects to special WebSocket which allows get information from terminal on server side. The
    * terminal is initialized only when the method is called the first time.
    */
```

Comment:
Could you please add some information about "options" parameter?

Reply:
done